### DISCUSSION TITLE
Error: "libtool not working for C++"

### DISCUSSION THREAD

**Discussion by omecamtiv** [termux/termux-packages#7408] [Q&A]
I was trying to build `libt3widget` which is a part of Tilde Terminal Toolkit in Termux. But while compiling it ended with the Error. I checked the config.log and it shows the following:
```----------------------------------                                      18 Checking for working libtool C++ compile...                             19 Running: make -f .Makefile '.configltcxx.la'                            20 libtool --mode=compile --tag=CXX aarch64-linux-android-clang++ -fstack-    protector-strong -Oz  -c -o .configltcxx.lo .configltcxx.cc -shared     21 libtool: compile:  aarch64-linux-android-clang++ -fstack-protector-stro    ng -Oz -c .configltcxx.cc  -fPIC -DPIC -o .libs/.configltcxx.o          22 libtool --mode=link --tag=CXX aarch64-linux-android-clang++ -fstack-pro    tector-strong -Oz -L/data/data/com.termux/files/usr/lib -fopenmp -stati    c-openmp -Wl,--enable-new-dtags -Wl,--as-needed -Wl,-z,relro,-z,now -o     libconfigltcxx.la .configltcxx.lo   -rpath /usr/lib -shared             23 libtool: link: aarch64-linux-android-clang++  -fPIC -DPIC -shared -nost    dlib /home/builder/.termux-build/_cache/android-r20-api-24-v3/bin/../sy    sroot/usr/lib/aarch64-linux-android/24/crtbegin_so.o  .libs/.configltcx    x.o   -L/data/data/com.termux/files/usr/lib -L/home/builder/.termux-bui    ld/_cache/android-r20-api-24-v3/lib64/clang/8.0.7/lib/linux/aarch64 -L/    home/builder/.termux-build/_cache/android-r20-api-24-v3/bin/../lib/gcc/    aarch64-linux-android/4.9.x -L/home/builder/.termux-build/_cache/androi    d-r20-api-24-v3/bin/../lib/gcc/aarch64-linux-android/4.9.x/../../../../    aarch64-linux-android/lib/../lib64 -L/home/builder/.termux-build/_cache    /android-r20-api-24-v3/bin/../sysroot/usr/lib/aarch64-linux-android/24     -L/home/builder/.termux-build/_cache/android-r20-api-24-v3/bin/../sysro    ot/usr/lib/aarch64-linux-android -L/home/builder/.termux-build/_cache/a    ndroid-r20-api-24-v3/bin/../lib/gcc/aarch64-linux-android/4.9.x/../../.    ./../aarch64-linux-android/lib -L/home/builder/.termux-build/_cache/and    roid-r20-api-24-v3/bin/../sysroot/usr/lib -lc++ -lm -lc -lgcc -ldl /hom    e/builder/.termux-build/_cache/android-r20-api-24-v3/bin/../sysroot/usr    /lib/aarch64-linux-android/24/crtend_so.o  -fstack-protector-strong -Oz     -fopenmp -Wl,--enable-new-dtags -Wl,--as-needed -Wl,-z -Wl,relro -Wl,-    z -Wl,now   -fopenmp -Wl,-soname -Wl,libconfigltcxx.so -o .libs/libconf    igltcxx.so                                                              24 clang-12: error: no such file or directory: '/home/builder/.termux-buil    d/_cache/android-r20-api-24-v3/bin/../sysroot/usr/lib/aarch64-linux-and    roid/24/crtbegin_so.o'                                                  25 clang-12: error: no such file or directory: '/home/builder/.termux-buil    d/_cache/android-r20-api-24-v3/bin/../sysroot/usr/lib/aarch64-linux-and    roid/24/crtend_so.o'                                                    26 make: *** [.Makefile:27: libconfigltcxx.la] Error 1```

Any idea why the error is occuring?

**Comment by thunder-coding** [comment]
`nd roid/24/crtbegin_so`
See that extra space probably that's the problem. it's there at multiple places

**Reply by omecamtiv** [reply]
no the extra space appered after copying the error log.
